Day 1/DayOnePartTwo.py

```diff
@@ -0,0 +1,31 @@
+import sys
+sys.path.insert(0, "C:/Users/Callipolis/Desktop/Programming/AoC2/Advent-of-Code-2015")
+from InputReader import InputReader
+
+input_reader = InputReader("input1.txt")
+raw_input = input_reader.get_raw_input()
+
+from DayOnePartOne import FloorCounter
+
+class BasementIndexFinder(FloorCounter):
+    
+    def __init__(self, puzzle_input):
+        super().__init__(puzzle_input)
```

Comment:
Ahogy most nez ki a FloorCounter, ez a class szerintem hibas most. Eloszor megoldja a part one-t az __init__-ben, aztan utana (nem nullarol indulva) megprobalja megoldani a masodikat.

Reply:
En amugy harom fuggvenyt raknek a FloorCounter-be
- get_floor_count
- count_floors
- chage_floor

es a part one main.py-ben meghivnam a count_floors meg a get_floor_count-ot, a part two-ban meg addig hivogatnam a change_floor-t amig a current_floor nem -1. 

(majd az inheritance-rol irok meg egy comment-et ha van idom)

Reply:
Köszi, kiganéztam a felesleges függvényeket a FloorCounterben. Kicsit most jobban átlátom az inheritance-ot az alapján, hogy miket futtatott le a konstruktorba bepakoltakból. Várom a kommentet.

Reply:
Most már látom, hogy ott van a komment alább.